thorlcr/thorutil/thormisc.hpp

```diff
@@ -321,6 +321,7 @@ class graph_decl CStreamFileOwner : public CSimpleInterfaceOf<IExtRowStream>
 #define DEFAULT_THORSLAVEPORT 20100
 #define DEFAULT_SLAVEPORTINC 20
 #define DEFAULT_QUERYSO_LIMIT 10
+#define DEFAULT_LINGER_SECS 60
```

Comment:
Feels a little long - we would expect next graph to come fairly immediately if it's coming, wouldn't we? I don't know what it costs to leave a cloud thor up for an extra minute, and I suppose in normal usage you'll tell it not to stay up unless there IS another graph coming, but...

Reply:
It is signalled at end of job to stop regardless.

I don't know how long is normal really, every job is different, it's not unusual though for graph1 to run, then for it to spend sometime performing e.g. superfile operations, or blocked on some locked file for a short while (sometimes a long while), or e.g. perform a spray.